source/fundamentals/connection/socks.txt

```diff
@@ -0,0 +1,139 @@
+.. _node-connect-socks:
+
+===========================
+Enable SOCKS5 Proxy Support
+===========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to connect to MongoDB instances with
+SOCKS5 proxy support. SOCKS5 is a standardized protocol for connecting
+to network services through a separate proxy server.
+
+.. tip::
+   
+   To learn more about SOCKS, see the Wikipedia entry on
+   :wikipedia:`SOCKS <SOCKS>`.
+
+Load the socks Module
+---------------------
+
+Starting in version 6.0 of the {+driver-short+}, you must *manually* load
+the ``socks`` dependency to use SOCKS5 proxy support in your
+application. You can install ``socks`` by running the following command
+in your terminal:
+
+.. code-block:: bash
+   
+   npm i socks
+
+SOCKS5 Client Options
+---------------------
+
+You can set options in your ``MongoClientOptions`` instance or
+parameters in your connection URI to configure SOCKS5 proxy support for 
+your connection. The following table describes the client options
+related to SOCKS5.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 1 1 1 1
+
+   * - Name
+     - Accepted Values
+     - Default Value
+     - Description
+
+   * - **proxyHost**
+     - string
+     - ``null``
+     - Specifies the IPv4/IPv6 address or domain name on which a SOCKS5
+       proxy is listening.
```

Comment:
Issue:

From [Slashes](https://www.mongodb.com/docs/meta/style-guide/style/punctuation/) in the style guide:
> Don't use a slash mark (/) to present a choice among, or a series of, actions or objects. Rewrite the phrase to eliminate the slash mark. Exceptions are established terms like client/server and read/write.

I think it could also be helpful to remove the perspective of the "proxy" which is "listening" since the configuration specifies behavior for the driver.

Suggestion:


```suggestion
     - Specifies the SOCKS5 proxy IPv4 address, IPv6 address, or domain name.
```